### PR TITLE
Add support for new gamma (auth.chalmers.it)

### PR DESCRIPTION
**Pull request by ViddeM**

### CODE REVIEW

File: backend/src/repositories/user.rs [hunk 1]
```diff
@@ -8,32 +9,54 @@ use crate::{
...
+      HubbitError::GammaError(err)
+    })?;
+
+    let groups = self.gamma_client.get_groups_for_user(&user.id).await?;
+
+    Ok(GammaUser::from_user_and_groups(user, groups))
+  }
+
+  pub async fn get_by_cid(&self, cid: &str) -> HubbitResult<GammaUser> {
+    log::warn!(
```
Comment (ViddeM) [inline backend/src/repositories/user.rs:36]: This is only used as a fallback and I'm not sure if it ever occurs in a real-world use-case (unless someone manually removes it from the cache).

File: backend/Cargo.toml [hunk 2]
```diff
@@ -43,3 +44,4 @@ tokio = { version = "1.36.0", features = [
     "time",
 ] }
 uuid = { version = "1.7.0", features = ["serde"] }
+gamma_rust_client = { git = "https://github.com/viddem/gamma.git", rev = "1235842562c9f3b50a68582dea13f66fbe93cf5e" }
```
Comment (Oscariremma) [inline backend/Cargo.toml]: Small TODO here. We should switch to the cthit hosted gamma repo and not a fork when the [gamma pr](https://github.com/cthit/Gamma/pull/880) is merged

**Review by Oscariremma — APPROVED** [review]
LGTM

File: backend/Cargo.toml (hunk 2 above)
Comment (ViddeM) [reply backend/Cargo.toml]: Done!

**Pull request merged**